### ISSUE TITLE
build error ubuntu 15.10

### ISSUE THREAD

**Issue by luntik2012** [papyros/qml-material#394]
I've built qt from git. I can not build qml-material with qt 5.5.0, qt 5.6.0, qt 5.6.1, qt 5.7.0 getting various errors like this(for 5.6.0):

> CMake Error at CMakeLists.txt:33 (find_package):
>   Could not find a package configuration file provided by "Qt5" (requested
>   version 5.4.0) with any of the following names:
> 
> ```
> Qt5Config.cmake
> qt5-config.cmake
> ```
> 
>   Add the installation prefix of "Qt5" to CMAKE_PREFIX_PATH or set "Qt5_DIR"
>   to a directory containing one of the above files.  If "Qt5" provides a
>   separate development package or SDK, be sure it has been installed.
> 
> -- Configuring incomplete, errors occurred!
##

**Comment by emmatyping** [comment]
I am getting this issue too.

At first it didn't find ECM:

```
CMake Error at CMakeLists.txt:13 (find_package):
  Could not find a package configuration file provided by "ECM" (requested
  version 0.0.11) with any of the following names:

    ECMConfig.cmake
    ecm-config.cmake

  Add the installation prefix of "ECM" to CMAKE_PREFIX_PATH or set "ECM_DIR"
  to a directory containing one of the above files.  If "ECM" provides a
  separate development package or SDK, be sure it has been installed.


-- Configuring incomplete, errors occurred!
```

Which I solved with [this SO answer](http://superuser.com/a/912259). However, now I am getting similar to what @luntik2012 is getting. Also trying to install from git.

**Comment by iBelieve** [comment]
We're dropping CMake to go back to QMake so the project will work with QPM. This will be fixed by #396.

**Comment by luntik2012** [comment]
Ubuntu 16.04
Qt 5.6.1

```
git clone https://github.com/papyros/qml-material
cd qml-material/
mkdir build
cd build/
qmake ..
make
```

```
/usr/local/Qt-5.6.1/bin/rcc -name demo ../../demo/demo.qrc -o qrc_demo.cpp
RCC: Error in '../../demo/demo.qrc': Cannot find file 'icons/navigation_menu.svg'
Makefile:561: recipe for target 'qrc_demo.cpp' failed
make[1]: *** [qrc_demo.cpp] Error 1
make[1]: Leaving directory '/usr/local/src/qml-material/build/demo'
Makefile:69: recipe for target 'sub-demo-make_first' failed
make: *** [sub-demo-make_first] Error 2

```
